packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx

```diff
@@ -91,9 +92,22 @@ class UtilityMenuItem extends React.Component {
     }
   }
 
+  handleIsSelectable() {
+    if (this.props.isSelectable) {
+      this.setState({ isSelected: !this.state.isSelected });
```

Comment:
Is this setting the state to deselected?

Reply:
No, it's toggling the `isSelected` state